api/specs/web-server/_storage.py

```diff
@@ -167,3 +174,49 @@ async def is_completed_upload_file(
     location_id: LocationID, file_id: StorageFileIDStr, future_id: str
 ):
     """Returns state of upload completion"""
+
+
+# data export
+@router.post(
+    "/storage/locations/{location_id}/export-data",
+    response_model=Envelope[StorageAsyncJobGet],
+    name="storage_export_data",
+    description="Export data",
+)
+async def export_data(data_export: DataExportPost, location_id: LocationID):
+    """Trigger data export. Returns async job id for getting status and results"""
+
+
+@router.get(
+    "/storage/async-jobs/{job_id}/status",
+    response_model=Envelope[StorageAsyncJobStatus],
+    name="storage_async_job_status",
+)
+async def get_async_job_status(storage_async_job_get: StorageAsyncJobGet, job_id: UUID):
+    """Get async job status"""
+
+
+@router.post(
+    "/storage/async-jobs/{job_id}:abort",
+    name="abort_async_job",
+)
+async def abort_async_job(storage_async_job_get: StorageAsyncJobGet, job_id: UUID):
+    """Get async job status"""
+
+
+@router.get(
+    "/storage/async-jobs/{job_id}/result",
+    response_model=Envelope[StorageAsyncJobResult],
+    name="get_async_job_result",
+)
+async def get_async_job_result(storage_async_job_get: StorageAsyncJobGet, job_id: UUID):
+    """Get async job status"""
+
+
+@router.get(
+    "/storage/async-jobs",
+    response_model=Envelope[list[StorageAsyncJobGet]],
+    name="get_async_job_result",
+)
+async def get_async_jobs():
+    """Get async job status"""
```

Comment:
The UserID is also missing below.
```suggestion
@router.get(
    "/storage/async-jobs",
    response_model=Envelope[list[StorageAsyncJobGet]],
    name="get_async_jobs",
)
async def get_async_jobs(user_id: UserID):
    """Retrunsa list of async jobs for the user"""
```

Reply:
Thanks